print-802_11.c

```diff
@@ -914,40 +914,45 @@ static const char *reason_text[] = {
 	"Reserved",						/* 0 */
 	"Unspecified reason",					/* 1 */
 	"Previous authentication no longer valid",		/* 2 */
-	"Deauthenticated because sending station is leaving (or has left) "
+	"Deauthenticated because sending STA is leaving (or has left)
```

Comment:
C:\projects\tcpdump\print-802_11.c(917): error C2001: newline in constant

Combine lines 917 and 918.